tools/jdbc/src/main/java/org/duckdb/DuckDBResultSet.java

```diff
@@ -155,6 +155,8 @@ public Object getObject(int columnIndex) throws SQLException {
 			return getDate(columnIndex);
 		case TIMESTAMP:
 			return getTimestamp(columnIndex);
+		case TIMESTAMP_WITH_TIME_ZONE:
+			return getTimestamp(columnIndex);
```

Comment:
I'm not horribly familiar with Java's Time APIs, but should this be `getOffsetDateTime`?

Reply:
You are right. Also a test for this getObject variant was missing.